src/amo/reducers/addonsByAuthors.js

```diff
@@ -1,125 +1,146 @@
 /* @flow */
-import type { AddonType, ExternalAddonType } from 'core/types/addons';
+import deepcopy from 'deepcopy';
+import invariant from 'invariant';
+
 import { createInternalAddon } from 'core/reducers/addons';
+import type { AddonType, ExternalAddonType } from 'core/types/addons';
 
 
 type State = {
+  byAddonId: { [number]: Array<AddonType> },
   byAddonSlug: { [string]: AddonType },
```

Comment:
It would be helpful to a todo comment about converting this part of the state later

Reply:
Actually I converted it but flow didn't catch it 🤷‍♂️. I've updated the type.

Reply:
Huh, weird. If I change the type back to `{ [string]: AddonType }` in your latest patch I definitely see a Flow error:

```
Error: src/amo/reducers/addonsByAuthors.js:92
 92:   return ids ? ids.map((id) => {
                        ^^^ property `map`. Property not found in
 92:   return ids ? ids.map((id) => {
                    ^^^ object type


Found 1 error
```